docs/integrations/destinations/snowflake.md

```diff
@@ -183,6 +183,16 @@ Field | Description |
 | [JDBC URL Params](https://docs.snowflake.com/en/user-guide/jdbc-parameters.html) (Optional) | Additional properties to pass to the JDBC URL string when connecting to the database formatted as `key=value` pairs separated by the symbol `&`. Example: `key1=value1&key2=value2&key3=value3` |
 
 
+### Key pair authentication
+    You need to generate private key without encryption - openssl genrsa 2048 | openssl pkcs8 -topk8 -inform PEM -out rsa_key.p8 -nocrypt
+
+    You can generate private key with encryption - openssl genrsa 2048 | openssl pkcs8 -topk8 -inform PEM -v1 PBE-SHA1-RC4-128 -out rsa_key.p8
+
+    Based on private key we need to generate public key with - openssl rsa -in rsa_key.p8 -pubout -out rsa_key.pub
+    Add generated public key to snowflake user - alter user <user_name> set rsa_public_key=<public_key_value>;
```

Comment:
Replace "Add generated public key to snowflake user" with "Finally, you need to add the public key to your Snowflake user account". You can do so with the following SQL command in Snowflake: `alter user <user_name> set rsa_public_key=<public_key_value>;` (replace <user_name> with your user name and <public_key_value> with your public key).